tests/fixtures/joint.py

```diff
@@ -293,51 +292,68 @@ class CleanupFixture:
 
     mongodb_dao: DrsObjectDaoPort
     joint: JointFixture
-    cached_id: str
-    expired_id: str
+    cached_file_ids: list[str]
+    expired_file_ids: list[str]
 
 
 @pytest_asyncio.fixture
 async def cleanup_fixture(
     joint_fixture: JointFixture,
 ) -> AsyncGenerator[CleanupFixture, None]:
     """Set up state for and populate CleanupFixture"""
-    # create AccessTimeDrsObjects for valid cached and expired cached file
-    test_file_cached = EXAMPLE_FILE.model_copy(deep=True)
-    test_file_cached.file_id = "cached"
-    test_file_cached.object_id = "cached"
-    test_file_cached.last_accessed = utc_dates.now_as_utc()
-
-    test_file_expired = EXAMPLE_FILE.model_copy(deep=True)
-    test_file_expired.file_id = "expired"
-    test_file_expired.object_id = "expired"
-    test_file_expired.last_accessed = utc_dates.now_as_utc() - timedelta(
-        days=joint_fixture.config.cache_timeout
-    )
-
-    # populate DB entries
+    # create common db dao to insert test data
     mongodb_dao = await joint_fixture.mongodb.dao_factory.get_dao(
         name="drs_objects",
         dto_model=models.AccessTimeDrsObject,
         id_field="file_id",
     )
-    await mongodb_dao.insert(test_file_cached)
-    await mongodb_dao.insert(test_file_expired)
-
-    # populate storage
-    with temp_file_object(
-        bucket_id=joint_fixture.bucket_id,
-        object_id=test_file_cached.object_id,
-    ) as cached_file:
+    cached_file_ids = []
+    expired_file_ids = []
+
+    for s3, file in (
+        (joint_fixture.s3, EXAMPLE_FILE),
+        (joint_fixture.second_s3, EXAMPLE_FILE_2),
+    ):
+        # create AccessTimeDrsObjects for valid cached and expired cached file
+
+        cached_file_id = file.file_id + "_cached"
+        cached_object_id = file.object_id + "-cached"
```

Comment:
If this is because underscores aren't allowed for the object storage, a str.replace("_", "-") might prevent wasted debug time if someone down the road decides the second file should be named second_test. 
Otherwise ignore me

Reply:
Jup, that's for the object storage ID. I'll adjust that

Reply:
On second thought, this is clearly traceable to the corresponding `EXAMPLE_FILE` and something hexkit will complain about with a clear error message, so I'd just leave it as is